### PR TITLE
Remove dependency on `package:charcode`.

### PR DESCRIPTION
**Pull request by lrhn**

### CODE REVIEW

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
Do you want to make this `0.3.5` so we can go straight to publish?

**Comment by lrhn** [comment]
I'd be fine with that. Fixing,.

**Comment by kevmoo** [comment]
also in changelog?

File: lib/src/charcodes.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+/// "Line feed" control character.
+const int $lf = 0x0a;
+
+/// Space character.
+const int $space = 0x20;
+
+/// Character `"`, short name.
+// ignore: constant_identifier_names
+const int $double_quote = 0x22;
```
Comment (kevmoo) [inline lib/src/charcodes.dart:20]: Why not `doubleQuote`? Seems silly to dance around our lint.

File: lib/src/charcodes.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+/// Character `#`.
+const int $hash = 0x23;
+
+/// Character `$`.
+const int $dollar = 0x24;
+
+/// Character "'".
+// ignore: constant_identifier_names
```
Comment (kevmoo) [inline lib/src/charcodes.dart:29]: ditto

File: lib/src/charcodes.dart (hunk 1 above)
Comment (lrhn) [reply lib/src/charcodes.dart:20]: I didn't try to rename things, just introduce the original names here.
We could rename them now. Let's do that.

**Review by kevmoo — APPROVED** [review]

**Pull request merged**